src/reynir_correct/errtokenizer.py

```diff
@@ -2447,6 +2579,11 @@ def late_fix_merges(
                     suggest=token.txt,
                 ),
             )
+        # Remove all annotations and revert corrections for tokens in wordlist
+        if token.original and token.original.strip() in ignore_wordlist:
+            token.txt = token.original.strip()
+            token.remove_error()
```

Comment:
Think whether the remove_error() operation (or something with a similar name) should take the new token text as a parameter and do the replacement and removal in one atomic operation. That could be more future-proof.

Reply:
Makes sense, done